cranelift/codegen/src/machinst/vcode.rs

```diff
@@ -219,6 +219,9 @@ pub struct EmitResult<I: VCodeInst> {
 
     /// Stack frame size.
     pub frame_size: u32,
+
+    /// The alignment requirement for pc-relative loads.
+    pub constant_alignment: u32,
```

Comment:
Can we call this just `alignment`? From a consumer point of view, the `EmitResult` represents a blob of machine code that has to be placed somewhere, with an alignment specified here; the fact that this alignment requirement comes from our constant pool in this instance is sort of an incidental detail (and there could be other reasons later).